data/en/videos.json

```diff
@@ -490,69 +587,78 @@
+      "description": "This short CDAP screencast shows you how you can quantize continuous values into groups, using CDAP's Data Prep feature. For more information on CDAP, ..."
     },
     {
       "videoId": "mSKwjKvYUtI",
       "preview": "https://i.ytimg.com/vi/mSKwjKvYUtI/hqdefault.jpg",
       "title": "#BDAM:  EDW Optimization with Hadoop and CDAP, by Sagar Kapare",
-      "visible": true
+      "visible": true,
+      "description": "Speaker: Sagar Kapare, Cask Big Data Applications Meetup, 05/10/2017 Palo Alto, CA More info here: http://www.meetup.com/BigDataApps/ Link to slides: ..."
     },
     {
       "videoId": "S6bRBzojObA",
       "preview": "https://i.ytimg.com/vi/S6bRBzojObA/hqdefault.jpg",
       "title": "#BDAM: Future proof, portable pipelines using Apache Beam, by Malo Denielou from Google",
-      "visible": true
+      "visible": true,
+      "description": "Speaker: Malo Denielou, Google Big Data Applications Meetup, 05/10/2017 Palo Alto, CA More info here: http://www.meetup.com/BigDataApps/ Link to slides: ..."
     },
     {
       "videoId": "j1DQ3FfduR0",
       "preview": "https://i.ytimg.com/vi/j1DQ3FfduR0/hqdefault.jpg",
       "title": "Jonathan Gray, Cask - Hadoop Summit 2016 San Jose #HS16SJ #theCUBE",
-      "visible": false
+      "visible": false,
+      "description": "Jonathan Gray speaks with John Furrier and George Gilbert from TheCube at Hadoop Summit 2016 in San Jose, CA. #HS16SJ #theCUBE."
     },
     {
       "videoId": "_tPix15gP3c",
       "preview": "https://i.ytimg.com/vi/_tPix15gP3c/hqdefault.jpg",
       "title": "Introducing Coopr",
-      "visible": false
+      "visible": false,
+      "description": "September 24, 2014."
     },
     {
       "videoId": "x4RM3l86ZgM",
       "preview": "https://i.ytimg.com/vi/x4RM3l86ZgM/hqdefault.jpg",
       "title": "Cask Intern Experience - Yue Gao, Summer 2016",
-      "visible": false
+      "visible": false,
+      "description": "Yue Gao interned at Cask for the summer of 2016, and here speaks about his summer internship experience. Yue is pursuing his Master's degree in Computer ..."
     },
     {
       "videoId": "735nEV7jxrs",
       "preview": "https://i.ytimg.com/vi/735nEV7jxrs/hqdefault.jpg",
       "title": "#BDAM: Unified access framework for distributed data systems on HDFS, by Shivram Mani",
-      "visible": true
+      "visible": true,
+      "description": "EXPAND FOR MORE INFO. Speaker: Shivram Mani, Pivotal Big Data Applications Meetup, 06/15/2016 Palo Alto, CA More info here: ..."
     },
     {
       "videoId": "t-FtGX_5UJI",
       "preview": "https://i.ytimg.com/vi/t-FtGX_5UJI/hqdefault.jpg",
       "title": "The DAP Where YARN, HBase, Kafka and Spark go to Production - by Jonathan Gray",
-      "visible": true
+      "visible": true,
+      "description": "June 30, 2016 Hadoop Summit '16, San Jose Title: The DAP Where YARN, HBase, Kafka and Spark go to Production Speaker: Jonathan Gray, CEO Cask ..."
```

Comment:
do we display description now?

Reply:
No we don't have description in design. Let me know if we need to show it somewhere.